code/online_challenges/src/hackerrank/solve_me_first.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+
+int solveMeFirst(int a, int b) {
+  int c;
+  c = a + b;
+  return c;
```

Comment:
```suggestion
  return a + b;
```

Also use 4 space indentation everywhere.